internal/cli/config.go

```diff
@@ -137,8 +137,7 @@ func (rc RunConfig) MaxTestsToRetryPercentage() (*float64, error) {
 
 func (rc RunConfig) IsRunningPartition() bool {
 	// TODO: Should we have a bit somewhere that indicates provider defaulted?
-	return rc.PartitionCommandTemplate != "" &&
-		(rc.PartitionConfig.PartitionNodes.Index != -1 || rc.PartitionConfig.PartitionNodes.Total != -1)
+	return rc.PartitionCommandTemplate != "" && rc.PartitionConfig.PartitionNodes.Total >= 1
```

Comment:
Probably fine, but really this probably only _practically_ makes sense if total is greater than 1 (2 or more, 1 partition is just the command itself? 🤔)

Reply:
That's a good point. I don't think it's necessarily wrong, but perhaps not what someone intended to do? Only scenario I can think of is if someone was like scripting their partition settings?

I think I'm inclined to get the fix in independently of this determination.

Reply:
I can see a use case for 1; imagine you have a script which detects changes in git and cobbles together only the tests you need to run based on your changes, and then dynamically sets the number of steps you have based on that. You might have a lot of times where you only want to run a couple of tests and so you only have 1 partition.

I'm not clear on how you would actually use our globbing mechanism to work with something like that, though 😅. I guess the most straightforward way would be that you only copy in the test files that you want to run, in which case it wouldn't matter whether we ran the command itself or if we ran a partition of 1.

Reply:
Yea I'm not worried about leaving this, it's mostly just a little extra work to get the timings, glob the files, etc when you could just call the command itself for the same effect. Definitely valid to _pass_ 1, just not obvious to me which behavior we should use 😅